docs/magics.rst

```diff
@@ -15,14 +15,14 @@ Code Samples
 
 Running a query:
 
-.. literalinclude:: ./samples/snippets/query.py
+.. literalinclude:: ../samples/snippets/query.py
```

Comment:
Does this actually work? In the past, I've had to put a symlink in the docs folder to the samples for the docs build to actually find the right files.